config/read_write_database_sample.php

```diff
@@ -0,0 +1,105 @@
+<?php
```

Comment:
This question is meant for @patrickbrouwers actually, as he suggested adding it :smile: